Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -131,15 +131,23 @@ private Loc GetLocationFromDevice()
         {
             // Location is ephemeral, so retrieve a new location from the platform. May return 0,0 which is
             // technically valid, though less likely real, as of writing.
+#if UNITY_EDITOR
+            Debug.Log("MobiledgeX: Cannot Get location in Unity Editor. Returning dummy location.");
+            return new Loc{
+                latitude = 37.3382082,
+                longitude = -121.8863286
+            };          
+#else
             Loc loc = LocationService.RetrieveLocation();
-            // If in UnityEditor, 0f and 0f are hard zeros as there is no location service.
+            // 0f and 0f are hard zeros if no location service.
```

Comment:
I would actually like to return 0's here, if that's going to be part of the Integration layer.

Reply:
I get this when plugging in 0,0: FindCloudletException : FindCloudlet Exception: Missing GpsLocation, HTTP StatusCode: BadRequest, API ErrorCode: 3

Reply:
May be a dme error that doesn't allow 0,0

Reply:
Right, how about a configurable fallback up top somewhere instead. It's 99.99% sure to be an error with 0,0, but at least they won't have to use our very specific location.

Reply:
so we fallback to this location struct if using UnityEditor or if LocationServices returns 0,0?

Reply:
Yeah, hope is not use it much outside the editor since there exists an example LocationService with permissions and everything.

Reply:
I think for now: I will let (0,0) slide, but print an error log. Then developer will probably receive the same FindCloudletException as I did and they will see that they are getting (0,0) location